inc/class-antivirus.php

```diff
@@ -350,10 +350,20 @@ public static function add_enqueue_style() {
 	/**
 	 * Get the currently activated theme.
 	 *
+	 * @param WP_Theme|false $t Theme to parse.
+	 *
 	 * @return array|false An array holding the theme data or false on failure.
+	 *
+	 * @since 1.4 Added $t parameter.
 	 */
-	private static function _get_current_theme() {
-		$theme = wp_get_theme();
+	private static function _get_current_theme( $t = false ) {
+		if ( $t ) {
```

Comment:
I would see variable names there describe his content, like $theme.

Reply:
`$theme` is already assigned one line below. I renamed the `$t` parameter to `$theme` now and re-assign it if `false`, instead of using two variables.